tables/output_tables.Rmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "Tables output for manuscript"
+author: "Run Jin"
+date: "10/27/2021"
+output:
+  pdf_document: default
+  html_document: default
+---
+
+```{r setup, include=FALSE}
+
+library(tidyverse)
+
+```
+
+## Output Tables for OpenPBTA Manuscript
+This is a Rmd files that record scripts for generating tables 
+
+```{r define directories}
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+working_dir <- file.path(root_dir, "tables")
+data_dir <- file.path(root_dir, "data")
```

Comment:
Based on the docs, I think we might want to tie this specifically to a release:

```suggestion
data_dir <- file.path(root_dir, "data", "release-v21-20210820")
```

That means, though, that we could not run this in CI but that is not something we're currently doing.